rpc/handle_get_info.go

```diff
@@ -0,0 +1,24 @@
+package rpc
+
+import (
+	"github.com/kaspanet/kaspad/rpc/model"
+	"github.com/kaspanet/kaspad/version"
+)
+
+// handleGetInfo implements the getInfo command. We only return the fields
+// that are not related to wallet functionality.
+func handleGetInfo(s *Server, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
+	ret := &model.InfoDAGResult{
```

Comment:
If there are any fields that are left TODO - please create a ticket

Reply:
There aren't.